sphinx/api/types.rst

```diff
@@ -89,18 +89,18 @@ data's column names in their SCADA data, plus the "freq" field. Users just have
 columns are already using the specified units, and that each column is already using the listed
 data type or can be converted to that type.
 
-==================== ==================================   =============================
- Field Name           Data Type (SCADAMetaData.dtypes)     Units (SCADAMetaData.units)
-==================== ==================================   =============================
- time                 datetime64[ns]                       datetime64[ns]
- id                   string                               None
- power                float                                kW
- windspeed            float                                m/s
- winddirection        float                                degrees
- status               string                               None
- pitch                float                                degrees
- temp                 float                                Celsius
-==================== ==================================   =============================
+==================== ================= ================================== =============================
+ Field Name          Descriptive Name   Data Type (SCADAMetaData.dtypes)   Units (SCADAMetaData.units)
+==================== ================= ================================== =============================
+ time                 time stamp        datetime64[ns]                     datetime64[ns]
+ id                   id                string                             None
+ WTUR_W               power             float                              kW
+ WMET_HorWdSpd        windspeed         float                              m/s
+ WMET_HorWdDir        winddirection     float                              degrees
+ status               status            string                             None
+ WROT_BlPthAngVal     pitch             float                              degrees
+ WMET_EnvTmp          temp              float                              Celsius
+==================== ================= ================================== =============================
```

Comment:
Nice table

Reply:
Thanks, this is really just an expansion of what we were already using to help make that naming switch a little bit easier.